app/jobs/models-utils.js

```diff
@@ -166,10 +176,139 @@ var job = {
                 message: 'error while fetching Jira Tickets',
                 error: ex.getMessage(),
             });
-            return "error occurred while getting jira link";
+            return "error occurred while getting jira Tickets";
         }
 
     },
+    sendEmailMessage: async function (emailConfig) {
+
+        try {
+            var SMTPConfig = emailConfig.SMTPConfig;
+            var assign = Object.keys(emailConfig.assign);
+            this.sendMail(assign, "assign", emailConfig["assign"], SMTPConfig);
+
+            var createdBy = Object.keys(emailConfig.assign);
+            this.sendMail(createdBy, "created by ", emailConfig["createdBy"], SMTPConfig);
+
+        } catch (error) {
+            logger.log({
+                level: 'error',
+                message: 'error occurred while sending notification of open tickets',
+                error: error,
+            });
+        }
+    },
+    sendMail: async function (userList, userType, tickets, SMTPConfig) {
+
+        var transporter = nodemailer.createTransport({
+            host: SMTPConfig.records.config.host,
+            port: SMTPConfig.records.config.port,
+            pool: true,
+            secure: false,
+            auth: {
+                user: SMTPConfig.records.config.user,
+                pass: SMTPConfig.records.config.password
+            },
+            tls: {
+                rejectUnauthorized: false
+            }
+        });
+
+        var mailOptions = {
+            from: SMTPConfig.records.config.sender,
+            to: null,
+            subject: null,
+            html: ""
+        };
+        userList.forEach(function (element) {
+            if (element !== "") {
+                mailOptions.subject = "Action required - Open Flair BI Alert Jira's - " + moment(moment().format()).format(dateFormat);
```

Comment:
create a separate yml file for subject,body and any constant should go into that file so any time it can be changed